protected/humhub/components/behaviors/AccessControl.php

```diff
@@ -220,4 +223,14 @@ protected function loginRequired()
 
         return false;
     }
+
+    /**
+     * @return Response Redirect user to force to change password
+     */
+    protected function mustChangePassword()
```

Comment:
Maybe rename to `forceChangePassword` for me there are a bit too many new `mustChangePassword` functions :)

Reply:
ok, commit https://github.com/humhub/humhub/pull/4602/commits/6444ceb70ce37b9a2b8251407d4c491edb0ccbf8